back/src/bsffs/resolvers/queries/__tests__/bsff.integration.ts

```diff
@@ -90,6 +114,30 @@ describe("Query.bsff", () => {
     ]);
   });
 
+  it("should allow admin user even if the user is not a contributor of the bsff", async () => {
```

Comment:
C'est clairement nice to have mais pas acquis de conscience j'aurais presque rajouté des tests pour vérifier qu'un utilisateur **non admin** étranger au BSD ne peut pas le get